lib/uart_blob/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_library()
+zephyr_library_sources_ifdef(CONFIG_UART_BLOB_TX uart_blob_tx.c)
+zephyr_library_sources_ifdef(CONFIG_UART_BLOB_RX uart_blob_rx.c)
+zephyr_include_directories(.)
```

Comment:
check if this is how the other `libs` does it, or if you should move the header file to the `nrf/include/` folder structure